spec/teacher_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe Teacher do
+  # The teacher should inherit from Person
+  it 'Inherit from Person' do
+    expect(Teacher.superclass).to eq Person
+  end
+
+  # Instantiate the Teacher class.
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 everything look fine congrats 🥳  Just one thing miss, according to the project requirements, there should be a unit tests for all the methods in all the classes, Kindly make sure the `can_use_services?` class method has (at least) an 'it' block 👇🏻 check your teacher,rb file

```ruby
  # Method can_use_services?
  def can_use_services?
    true
  end
```